ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3,8 +3,9 @@
 from types import SimpleNamespace
 import numpy as np
 
-from ivy_tests.test_ivy.test_frontends.test_torch.test_comparison_ops import \
-    _topk_helper
+from ivy_tests.test_ivy.test_frontends.test_torch.test_comparison_ops import (
+    _topk_helper,
+)
```

Comment:
Can we revert back this modification?

Reply:
Hi, this was also due to automatic pre-commit that we have.

Reply:
Oh okay, we'll leave it then